lib/zendesk_apps_support/manifest/no_override_hash.rb

```diff
@@ -18,7 +18,7 @@ def message
 
           # if the error contains the word `_legacy` in the second sentence, let's
           # only use the first one.
-          if [original, attempted].any? { |val| val =~ /_legacy/ }
+          if [original, attempted].any? { |val| val.is_a?(String) && val =~ /_legacy/ }
```

Comment:
`=~` for non-strings has been removed from recent Ruby versions (and was deprecated—and often meaningless—before), so now we compare only strings to this regex.